packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap

```diff
@@ -0,0 +1,1918 @@
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 8,
+    },
+    "value": -128,
+  },
+  {
+    "name": "int8_2D",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 1,
+      "total_size": 6,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [
+      2,
+      3,
+    ],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 8,
+    },
+    "value": Int8Array [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+    ],
+  },
+  {
+    "name": "int16_scalar",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 2,
+      "total_size": 1,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 16,
+    },
+    "value": -32768,
+  },
+  {
+    "name": "int16_2D",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 2,
+      "total_size": 6,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [
+      2,
+      3,
+    ],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 16,
+    },
+    "value": Int16Array [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+    ],
+  },
+  {
+    "name": "int32_scalar",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 4,
+      "total_size": 1,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 32,
+    },
+    "value": -2147483648,
+  },
+  {
+    "name": "int32_BE_scalar",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": false,
+      "signed": true,
+      "size": 4,
+      "total_size": 1,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [],
+    "type": {
+      "class": "Integer",
+      "endianness": "big-endian",
+      "size": 32,
+    },
+    "value": 0,
+  },
+  {
+    "name": "int32_2D",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 4,
+      "total_size": 6,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [
+      2,
+      3,
+    ],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 32,
+    },
+    "value": Int32Array [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+    ],
+  },
+  {
+    "name": "int64_scalar",
+    "rawType": {
+      "cset": -1,
+      "littleEndian": true,
+      "signed": true,
+      "size": 8,
+      "total_size": 1,
+      "type": 0,
+      "vlen": false,
+    },
+    "shape": [],
+    "type": {
+      "class": "Integer",
+      "endianness": "little-endian",
+      "size": 64,
+    },
+    "value": -9223372036854776000,
```

Comment:
The exact value in the file is -9,223,372,036,854,775,808, so there's a bit of rounding happening somewhere. There are other cases like this of course.